.github/workflows/version-bump.yml

```diff
@@ -0,0 +1,126 @@
+name: 'Version Bump'
+on:
+  workflow_dispatch:
+    inputs:
+      release_line:
+        description: 'Release Line'
+        required: true
+        default: next
+        type: choice
+        options:
+          - next
+          - main
+
+  # Run Weekly on Wednesday at 10:00 UTC time
+  schedule:
+    - cron: '0 10 * * 3'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  version-bump:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        workspace: ['azure-devops', 'linguist']
```

Comment:
Ideally this would be great if we could automate this list. Hard coded for now to test

@benjdlambert feel like you had some ideas here?

Reply:
So I realised because the workspace don't already have changes then the existing script isn't going to work:

I'd probably recommend creating a `scripts/list-workspaces.js` that can do a simple `readDir` and output the available workspaces using the same approach as [the other script](https://github.com/backstage/community-plugins/blob/main/scripts/ci/list-workspaces-with-changes.js)

Then you should be able to use the output like the [current workflows](https://github.com/backstage/community-plugins/blob/main/.github/workflows/ci.yml#L44) do.

Reply:
Will look into this in a follow up PR 👍